docs/guides/how-do-i-use-orbstack.md

```diff
@@ -0,0 +1,48 @@
+---
+title: Using Lando with Orbstack
+description: Learn how to use Lando with Orbstack today.
+guide: true
+
+authors:
+  - name: Oscar Arzola
+    pic: https://avatars.githubusercontent.com/u/4997549?v=4
+    link: https://twitter.com/init_sh
+updated:
+  timestamp: 1733175010
+---
+
+# Using Lando with Orbstack
+
+[Orbstack](https://docs.orbstack.dev/) is a lightweight and ⚡️ lightning-fast Docker provider that can serve as a drop-in replacement for Docker Desktop. It is an excellent alternative for users who want to use Lando without Docker Desktop.
+
+Official support for swapping Docker providers will be available in [Lando 4](https://lando.dev/blog/2023/01/23/roadmap-of-2023.html). In the meantime, you can use [Orbstack](https://orbstack.dev/) as a drop-in replacement.
+
+After installing Lando, follow these steps:
+
+1. Kill all running containers, close Docker Desktop and ensure that auto-start is disabled in Docker Desktop settings.
```

Comment:
Maybe include commands here to do this... `lando poweroff && docker stop $(docker ps -a -q)`, etc..

Reply:
this is a great idea will do it